tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -35,6 +36,8 @@ Verify Openvino Operator Can Be Installed Using OpenShift Console
    Go To RHODS Dashboard
    Verify Service Is Enabled          ${openvino_container_name}
    Verify JupyterHub Can Spawn Openvino Notebook
+   Clone Git Repository      https://github.com/sclorg/nodejs-ex.git
+   Verify File Present In The File Explorer      nodejs-ex
```

Comment:
since you added these lines, the teardown should also clean up the Jupyter environment

Reply:
I have added Clean Up Server in every test-case teardown which will delete the clone repository.